### PR TITLE
docs: direct settings object format

### PR DESCRIPTION
**Pull request by hpohekar**
closes https://github.com/ansys/pyfluent/issues/3416

Updated docs for direct settings object format.

image

### CODE REVIEW

File: doc/source/user_guide/solver_settings/index.rst [hunk 1]
```diff
@@ -2,28 +2,53 @@
...
+who prefer the existing hierarchy.
+
+.. code-block:: python
+
   >>> file = solver.settings.file
   >>> setup = solver.settings.setup
   >>> solution = solver.settings.solution
   >>> results = solver.settings.results
 
-Note that the last three are top-level nodes in the outline tree view in Fluent's graphical
```
Comment (seanpearsonuk) [inline doc/source/user_guide/solver_settings/index.rst:24]: @hpohekar Please could you also update the few remaining instances of the "old" syntax below. Also, we could change the heading "Supporting wildcards" to just "Wildcards".

**Comment by seanpearsonuk** [comment]
@hpohekar Maybe look into this issue?

image

**Comment by hpohekar** [comment]
> @hpohekar Maybe look into this issue?
> 
> image

@seanpearsonuk It will be available once settings doc are also available after nightly dev doc build.

**Comment by seanpearsonuk** [comment]
> > @hpohekar Maybe look into this issue?
> > image
> 
> @seanpearsonuk It will be clickable once settings doc are also available after nightly dev doc build.

@hpohekar I see, thanks for clarifying.

**Review by seanpearsonuk — APPROVED** [review]

File: doc/source/user_guide/solver_settings/index.rst (hunk 1 above)
Comment (hpohekar) [reply doc/source/user_guide/solver_settings/index.rst:24]: Done. Thank you.

File: doc/source/user_guide/solver_settings/set_up/models/viscous.rst [hunk 2]
```diff
@@ -6,7 +6,8 @@ Setting up and querying the model
 
 .. code:: python
 
-    >>> viscous = solver.settings.setup.models.viscous
+    >>> import ansys.fluent.core as pyfluent
+    >>> viscous = Viscous(settings_source=solver)
```
Comment (mkundu1) [inline doc/source/user_guide/solver_settings/set_up/models/viscous.rst:10]:
```suggestion
    >>> viscous = pyfluent.Viscous(settings_source=solver)
```

File: doc/source/user_guide/solver_settings/index.rst [hunk 3]
```diff
@@ -303,17 +349,21 @@ commands::
...
-Supporting wildcards
---------------------
+Wildcards
+---------
 You can use wildcards when using named objects, list objects, and string list settings.
 For named objects and list objects, for instance::
 
-  >>> solver.settings.setup.cell_zone_conditions.fluid["*"].source_terms["*mom*"]()
+  >>> import ansys.fluent.core as pyfluent
+  >>> fluid = pyfluent.FluidCellZone(settings_source=solver, name="*")
```
Comment (mkundu1) [inline doc/source/user_guide/solver_settings/index.rst:358]: Need to check if this is supported

File: doc/source/user_guide/solver_settings/set_up/models/viscous.rst (hunk 2 above)
Comment (hpohekar) [reply doc/source/user_guide/solver_settings/set_up/models/viscous.rst:10]: Done. Thank you.

File: doc/source/user_guide/solver_settings/index.rst (hunk 3 above)
Comment (mkundu1) [reply doc/source/user_guide/solver_settings/index.rst:358]: This works, thanks

**Review by mkundu1 — APPROVED** [review]

**Pull request merged**